src/web/screens/apiKeyManagement.tsx

```diff
@@ -72,7 +72,7 @@ function ApiKeyManagement() {
     <div className='api-key-management-page'>
       <h1>Manage API Keys</h1>
       <p className='heading-details'>
-        View and manage your API keys. For more information on API Roles, please visit our{' '}
+        View and manage your API keys. For more information, see API permissions.
```

Comment:
I think `API permissions` needs to move to line 81 to replace "docs"

Reply:
Ah yeah, my bad sorry.